src/test/java/teammates/test/cases/automated/FeedbackSessionClosedReminderTest.java

```diff
@@ -0,0 +1,183 @@
+        session1.setEndTime(TimeHelper.getDateOffsetToCurrentTime(0));
+        fsLogic.updateFeedbackSession(session1);
+        verifyPresentInDatastore(session1);
+       
+        // Reuse an existing session to create a new one that is closed.
+        FeedbackSessionAttributes session2 = dataBundle.feedbackSessions.get("session2InCourse2");
+        
+        session2.setTimeZone(0);
+        session2.setStartTime(TimeHelper.getDateOffsetToCurrentTime(-2));
+        session2.setEndTime(TimeHelper.getDateOffsetToCurrentTime(0));
+        fsLogic.updateFeedbackSession(session2);
+        verifyPresentInDatastore(session2);
+        
+        // Reuse an existing session to create a new one that is closed and closing reminder disabled.
+        FeedbackSessionAttributes session3 = dataBundle.feedbackSessions.get("session2InCourse1");
+        
+        session3.setTimeZone(0);
+        session3.setStartTime(TimeHelper.getDateOffsetToCurrentTime(-2));
+        session3.setEndTime(TimeHelper.getDateOffsetToCurrentTime(0));
+        session3.setClosingEmailEnabled(false);
+        fsLogic.updateFeedbackSession(session3);
+        verifyPresentInDatastore(session3);
+        
+        int counter = 0;
+        while (counter != 10) {
+            FeedbackSessionClosedCallback.resetTaskCount();
+            fsLogic.scheduleFeedbackSessionClosedEmails();
+            //There are only 2 sessions closed reminder to be sent
+            if (FeedbackSessionClosedCallback.verifyTaskCount(2)) {
+                break;
+            }
+            counter++;
+        }
+        if (counter == 10) {
+            assertEquals(FeedbackSessionClosedCallback.taskCount, 2);
+        }
+    }
+
+    @Test
+    public void testFeedbackSessionClosedMailAction() throws Exception {
+        
+        ______TS("typical case, testing mime messages");
+        // Modify session to closed.
+        FeedbackSessionAttributes session1 = dataBundle.feedbackSessions.get("session1InCourse1");
+        session1.setTimeZone(0);
+        session1.setStartTime(TimeHelper.getDateOffsetToCurrentTime(-1));
+        session1.setEndTime(TimeHelper.getDateOffsetToCurrentTime(0));
+        fsLogic.updateFeedbackSession(session1);
+        HashMap<String, String> paramMap = createParamMapForAction(session1);
+        String course1Name = CoursesLogic.inst().getCourse(session1.getCourseId()).getName();
+        
+        EmailAction fsClosedAction = new FeedbackSessionClosedMailAction(paramMap);
+        int course1InstructorCount = 5;
+        
+        List<EmailWrapper> preparedEmails = fsClosedAction.getPreparedEmailsAndPerformSuccessOperations();
+        assertEquals(course1InstructorCount, preparedEmails.size());
+        
+        for (EmailWrapper email : preparedEmails) {
+            assertEquals(String.format(EmailType.FEEDBACK_CLOSED.getSubject(), course1Name,
+                                       session1.getFeedbackSessionName()), email.getSubject());
```

Comment:
Add a line break at `email.getSubject()` here, like how it's done in many other places.